src/pages/messaging/agent-chat-api/changelog/index.mdx

```diff
@@ -21,13 +21,14 @@ The developer preview version provides a preview of the upcoming changes to the
 
 ## [v3.6] - Developer preview
 
-No changes yet.
+- The **Get Chat** ([Web](/messaging/agent-chat-api/v3.6/#get-chat) & [RTM](/messaging/agent-chat-api/v3.6/rtm-reference/#get-chat)) method now validates access to the requested thread.
```

Comment:
Hello :) Can we please do it the way it is done [here](https://developers.livechat.com/docs/messaging/agent-chat-api/changelog#chats) - I would add a heading - `Chats` in this case. I know that there is a PR created by Wojtek where changes are introduced in exactly the same place -> [here](https://github.com/livechat/livechat-public-docs/pull/1275). I was asked to wait with publishing Wojtek's PR, so I believe there might be some conflicts at some point between your PR and his. I will be honest - I have no clue if there is any way to avoid them and whose PR will be published first so I decided to let you know :)

Reply:
Hey, thanks, added the heading.
As to who goes first - me and @wfabjanczuk will butt our bald heads and whoever wins goes first :)

Reply:
@litwinow you are mr bald guy #1

Reply:
😂